### PR TITLE
Add test to do config reload and check for messages

### PR DESCRIPTION
**Pull request by judyjoseph**
Summary:
Fixes https://github.com/Azure/sonic-mgmt/issues/4245

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911

#### How did you do it?
Add a new test test_po_cleanup_after_reload, which checks if portchannels are removed during config reload.

Add a way to stress the device by making CPU busy when config reload is done.

#### How did you verify/test it?
```
jujoseph@1a78aa977682:/data/sonic-mgmt/tests$ ./run_tests.sh -n vms-kvm-t1-lag -d vlab-03 -c pc/test_po_cleanup.py::test_po_cleanup_after_reload -f vtestbed.csv -i veos_vtb -u -e "--skip_sanity"
=== Running tests in groups ===
/usr/local/lib/python2.7/dist-packages/ansible/parsing/vault/__init__.py:44: CryptographyDeprecationWarning: Python 2 is no longer supported by the Python core team. Support for it is now deprecated in cryptography, and will be removed in the next release.
  from cryptography.exceptions import InvalidSignature
=============================================================================================== test session starts ================================================================================================
platform linux2 -- Python 2.7.17, pytest-4.6.5, py-1.10.0, pluggy-0.13.1
ansible: 2.8.12
rootdir: /data/sonic-mgmt/tests, inifile: pytest.ini
plugins: metadata-1.11.0, repeat-0.9.1, forked-1.3.0, html-1.22.1, xdist-1.28.0, ansible-2.2.2
collecting ... ['conf-name', 'group-name', 'topo', 'ptf_image_name', 'ptf', 'ptf_ip', 'ptf_ipv6', 'server', 'vm_base', 'dut', 'inv_name', 'auto_recover', 'comment']
/usr/local/lib/python2.7/dist-packages/ansible/parsing/vault/__init__.py:44: CryptographyDeprecationWarning: Python 2 is no longer supported by the Python core team. Support for it is now deprecated in cryptography, and will be removed in the next release.
  from cryptography.exceptions import InvalidSignature
collected 1 item                                                                                                                                                                                                   

pc/test_po_cleanup.py::test_po_cleanup_after_reload[vlab-03] PASSED                                                                                                                                          [100%]

------------------------------------------------------------------------------ generated xml file: /data/sonic-mgmt/tests/logs/tr.xml ------------------------------------------------------------------------------
============================================================================================ 1 passed in 314.83 seconds ============================================================================================
```

### CODE REVIEW

File: tests/pc/test_po_cleanup.py [hunk 1]
```diff
@@ -62,3 +60,45 @@ def test_po_cleanup(duthosts, enum_rand_one_per_hwsku_frontend_hostname, enum_as
...
+        loganalyzer.expect_regex.append(LOG_EXPECT_PO_CLEANUP_RE.format(pc))
+
+    try:
+        # Make CPU high
+        for i in range(host_vcpus):
+            duthost.shell("nohup yes > /dev/null 2>&1 & sleep 1")
+
+        with loganalyzer:
+            logging.info("Reloading config..")
+            config_reload(duthost)
```
Comment (bingwang-ms) [inline tests/pc/test_po_cleanup.py:99]: I was thinking if all the CPUs are 100% occupied, can we lose access to the device? How can we stop all running `yes` if that happens?

Comment (judyjoseph) [reply tests/pc/test_po_cleanup.py:99]: We have the "sleep 1" which **prevents** the CPU from going 100% CPU and not responsive. I have verified it is ok.

**Review by bingwang-ms — APPROVED** [review]

**Pull request merged**